subdomains.md

```diff
@@ -6,15 +6,29 @@ First, choose a name. In this tutorial we'll use `example` which will be `exampl
 
 ## Galaxy Configuration
 
-1. [Add your site](https://github.com/usegalaxy-eu/infrastructure-playbook/blob/master/group_vars/custom-sites.yml). It should look something like:
+1. add a folder named like your subdomain (here: example.usegalaxy.com) in this [directory](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/files/galaxy/subdomains), with a subfolder called themes containing a yaml file named like your subdomain (if you are not using a theme yet, you can leave the file empty).
```

Comment:
This is not an optional step?
maybe a sentence like "Have a look at other subdomains to get inspiration to add custom themes or contact us for help".

Reply:
No this is not optional